packages/mdc-textfield/_mixins.scss

```diff
@@ -124,6 +124,12 @@
   }
 }
 
+@mixin mdc-text-field-caret-color($color) {
+  &:not(.mdc-text-field--disabled) {
```

Comment:
I think we can remove the `not` selector and the private mixin, since disabled text fields don't have a caret.